README.md

```diff
@@ -16,13 +16,6 @@ Additional description about the project and its features.
 - Frameworks: NO
 - Technologies used: Github and git flow, VS code
 
```

Comment:
- [ ] The attached screenshot of project on _line 9_ of your ReadMe does not reflect the screen of the actual project. Kindly ensure to improve upon this aspect.